README.md

```diff
@@ -14,8 +14,10 @@ Examples of how to utilize and integrate Chainlinks can be found in our [Hello C
 ## Install
 
 1. [Install Go 1.9+](https://golang.org/doc/install#install), and add your GOPATH's [bin directory to your PATH](https://golang.org/doc/code.html#GOPATH)
-2. Install ChainLink: `$ go get -u github.com/smartcontractkit/chainlink`
-3. Run the node: `$ chainlink help`
+2. Install [dep](https://github.com/golang/dep#installation): `$ brew install dep` <br> or `$ go get -u github.com/golang/dep/cmd/dep`
+3. Clone ChainLink: `$ git clone github.com/smartcontractkit/chainlink`
```

Comment:
Trying this out locally, the URL needs "https://" in front of it:

3. Clone ChainLink: `$ git clone https://github.com/smartcontractkit/chainlink`

Reply:
Ah, right. Got that updated in both places for cloning.